src/go/k8s/pkg/resources/statefulset.go

```diff
@@ -333,12 +333,10 @@ func (r *StatefulSetResource) obj() (k8sclient.Object, error) {
 							Args: []string{
 								"redpanda",
 								"start",
-								"--check=false",
-								"--smp 1",
 								// sometimes a little bit of memory is consumed by other processes than seastar
 								"--reserve-memory " + redpandav1alpha1.ReserveMemoryString,
 								r.portsConfiguration(),
-								"--default-log-level=debug",
```

Comment:
FYI I opened https://github.com/vectorizedio/redpanda/issues/1149

Reply:
Thanks will address that in near future.